mycelo/env/types.go

```diff
@@ -17,5 +17,5 @@ type Config struct {
 	InitialValidators  int      `json:"initialValidators"`  // Number of initial validators
 	ValidatorsPerGroup int      `json:"validatorsPerGroup"` // Number of validators per group in the initial set
 	DeveloperAccounts  int      `json:"developerAccounts"`  // Number of developers accounts
-
+	LoadTestTPS        int      `json:"loadTestTPS"`        // Transactions per second to target in the load test
```

Comment:
i don't get why we need this parameter, nor where this is used.

In any case, it doesn't seem to be a good fit for env.Config. I would put here things that are about the genessis block creation, accounts etc.... loadTestTPS seems to be a load-bot cmd only config variable

Reply:
Removed